CI/travis.before_install.sh

```diff
@@ -1,8 +1,4 @@
 #!/bin/bash
-if [ -z "${TRAVIS_OS_NAME}" ] || [ "${TRAVIS_OS_NAME}" = "linux" ]; then
-  echo Before install on linux.
```

Comment:
Linux builds will not be silent at some point now, you can clearly see every command Travis is running. It'll still install things and show the usual [apt get](https://travis-ci.org/Mudlet/Mudlet/jobs/220564750#L137) command. The gcc version is still printed as well (in fact it was [previously](https://travis-ci.org/Mudlet/Mudlet/jobs/220564750#L412) [duplicated](https://travis-ci.org/Mudlet/Mudlet/jobs/220564750#L421)). I don't believe we are losing any debugging information here.